File: CS3305/Demos/demo1/Java/testTimer.java
// To execute this program:
//     javac testTimer.java
//     java testTimer
//
public class testTimer {

    public static void main( String [] args ) {

        Timer myTimer = new Timer();

        System.out.printf("Setting timer for 5 seconds\n");
        myTimer.run_timer(5);

        System.out.printf("Timer ran for %s seconds\n", myTimer );

        System.exit(0);
    }
}

File: CS3305/Demos/demo1/Python/test_timer.py
#!/usr/bin/env python3
'''
    To execute this program:
    ./testTimer.py
'''

from timer import Timer


def main():
    '''simple program to test the timer'''

    print("Setting timer for 5 seconds")
    my_timer = Timer()
    my_timer.run_timer(5)

    print("Timer ran for " + str(my_timer) + " seconds")

    exit(0)
# end main


if __name__ == "__main__":
    main()
